command.go

```diff
@@ -930,6 +934,23 @@ func (c *Command) execute(a []string) (err error) {
 		argWoFlags = a
 	}
 
+	if c.OnKillRun != nil {
+		sigchan := make(chan os.Signal)
+		signal.Notify(
+			sigchan,
+			syscall.SIGINT,
+			syscall.SIGKILL,
+			syscall.SIGTERM,
+			syscall.SIGQUIT,
+		)
+
+		go func() {
+			_ = <-sigchan
+
+			c.OnKillRun(c, argWoFlags)
```

Comment:
I wonder if the function would be interested in knowing the signal that was received?
For example, `SIGQUIT` is a request for a core dump; the `OnKillRun` function may want to honour that.
What do you think of passing the received signal to the `OnKillRun` function?